src/services/tx/tx-sender/dispatch.ts

```diff
@@ -141,18 +141,19 @@ export const dispatchOnChainSigning = async (
   chainId: SafeInfo['chainId'],
   signerAddress: string,
   safeAddress: string,
+  isNestedSafe: boolean,
 ) => {
   const sdk = await getSafeSDKWithSigner(provider)
   const safeTxHash = await sdk.getTransactionHash(safeTx)
   const eventParams = { txId, nonce: safeTx.data.nonce }
 
   const options = chainId === chains.zksync ? { gasLimit: ZK_SYNC_ON_CHAIN_SIGNATURE_GAS_LIMIT } : undefined
-
+  let txResponse: any
```

Comment:
If you type this as a `string`, you don't need to reassign the type in `parentSafeTxHash`.